content/guides/development.md

```diff
@@ -240,7 +240,301 @@ T> Now that your server is working, you might want to give [Hot Module Replaceme
 
 ### Using webpack-dev-middleware
 
-?> Familiar with `webpack-dev-middleware`? We need your help! Please submit a PR to fill in the missing instructions and example here. Make sure to keep it simple as this guide is intended for beginners.
+`webpack-dev-middleware` is a wrapper that will emit files processed by webpack to a server. It's actually used in the `webpack-dev-server` itself, but if you want to use it with a different configuration (e.g. with `express`), you can use the middleqware.
+
+Let's install `express` and `webpack-dev-middleware` so we can get started:
+
+``` bash
+npm install --save-dev express webpack-dev-middleware
+```
+
+Now we need to make some adjustments to our webpack configuration file in order to make sure the middleware will function correctly:
+
+__webpack.config.js__
+
+``` diff
+  var path = require('path');
+  const htmlWebpackPlugin = require('html-webpack-plugin');
+  const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+  module.exports = {
+    entry: {
+      app: './src/index.js',
+      print: './src/print.js'
+    },
+    devtool: 'inline-source-map',
+    plugins: [
+      new CleanWebpackPlugin(['dist']),
+      new htmlWebpackPlugin({
+        title: 'Output Management'
+      }),
+      new webpack.HotModuleReplacementPlugin()
+    ],
+    output: {
+      filename: '[name].bundle.js',
+-     path: path.resolve(__dirname, 'dist')
++     path: '/',
++     publicPath: 'http://localhost:3000/app'
+    }
+  };
+```
+
+This will tell the middleware to serve our files on `http://localhost:3000/app`, but before we can actually see the files, we need to setup `express` and make sure it's running.
+
+__project__
+
+``` bash
+  webpack-demo
+  |- package.json
+  |- webpack.config.js
+  |- /dist
+  |- /src
+    |- index.js
+    |- print.js
++   |- server.js
+  |- /node_modules
+```
+
+__src/server.js__
+
+``` js
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('../webpack.config.js');
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(webpackDevMiddleware(compiler, {
+  publicPath: config.output.publicPath
+}));
+
+app.listen(3000, function () {
+  console.log('Example app listening on port 3000!');
+});
+```
+
+Now add an npm script to make it a little easier to run the server:
+
+__package.json__
+
+``` json
+  {
+    "name": "development",
+    "version": "1.0.0",
+    "description": "",
+    "main": "webpack.config.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1",
+      "watch": "webpack --progress --watch",
+      "start": "webpack-dev-server --open",
++     "server": "node src/server.js",
+      "build": "webpack"
```

Comment:
This diff looks a bit funny, not sure if you see it the same way?

![image](https://user-images.githubusercontent.com/8988697/28916893-b4968850-7811-11e7-8c5b-53b83a0c2800.png)

Is that indicating tabs were used instead of spaces or something?